components/http_foundation/introduction.rst

```diff
@@ -140,7 +140,7 @@ has some methods to filter the input values:
 :method:`Symfony\\Component\\HttpFoundation\\ParameterBag::filter`
     Filters the parameter by using the PHP :phpfunction:`filter_var` function.
 
-All getters takes up to three arguments: the first one is the parameter name
+All getters take up to two arguments: the first one is the parameter name
```

Comment:
So to be honest, I would rather favour to remove the description of that feature than to document it even more (but keeping it the way it is is okay for me too).